qir/qsharp/Atomics/AdvancedConstantFolding/Makefile

```diff
@@ -1,7 +1,7 @@
 all: qat-qsharp
 
 qat-qsharp: QSharpVersion/qir/Example.ll
-	${QAT_BINARY} --apply -S -O3 --verify-module --validate --always-inline --unroll-loops --profile default QSharpVersion/qir/Example.ll  || echo "failed!"
+	${QAT_BINARY} --apply -S --verify-module --validate --always-inline --unroll-loops --adaptor default QSharpVersion/qir/Example.ll  || echo "failed!"
```

Comment:
Suspicious diff. I expected

Reply:
I think this would be part of a larger refactor: We should rearrange the examples and move to a more unified build system. I've created an issue for this item #140 .

Reply:
I've checked the character codes which turns out to be 45, but just to be sure, I've replaced the double dash.